src/main.rs

```diff
@@ -128,10 +131,17 @@ pub fn list_languages(assets: &HighlightingAssets, term_width: usize) {
     }
 }
 
-pub fn list_themes(assets: &HighlightingAssets) {
+pub fn list_themes(assets: &HighlightingAssets, config: &mut Config) {
     let themes = &assets.theme_set.themes;
+    let mut style = HashSet::new();
+    style.insert(OutputComponent::Plain);
+    config.files = vec![Some("assets/theme_preview.rs")];
+    config.output_components = OutputComponents(style);
     for (theme, _) in themes.iter() {
-        println!("{}", theme);
+        println!("{}\n", Style::new().bold().paint(theme));
```

Comment:
It'd be great if we could make this work in Rust 1.26 (build currently fails)

Reply:
I'm using the most recent example from the docs ... if we want it to compile for anything other than stable Rust we may want to unbold the titles or look for another library (which would kinda be overkill)

Reply:
I think its just that `theme` needs to be casted explicitly ... I can look into this.